public/src/inverted-index.js

```diff
@@ -1,8 +1,9 @@
 // This program attempts to build an inverted index from a JSON file
 /**
-* @class InvertedIndex
-*
-*/
+ * Inverted Index
+ * @class
+ *
```

Comment:
You can make you documentation comments more complete and avoid the empty lines.